_data/students.yml

```diff
@@ -100,7 +100,11 @@
 - name: Kartikeya Singh
   github: kartikeya23
   image: kart.png
-
+  
+- name: Rahul Metre
+  github: RahulMetre03
+  image: icon.png
```

Comment:
Try something meaningful, ```icon.png``` relates to other images.

Reply:
@RahulMetre03 Please don't mark the requested changes as resolved when actually they are not yet resolved. Change the image name to ```RahulMetre.png``` or something like that. Then it would be good to go.